Fixing typos and missing instructions in Desktop and SPA Tutorials

Rich-Lang opened the pull request:
Adding Link/Button to Desktop & SPA tutorials to be consistent with WebApp tutorial.
Updating Next Step description typo in SPA tutorial.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

alexandair commented:
@mgoedtel The article mentions 3 parameters when actual runbook on PowerShell Gallery has only two parameters. The name of the runbook is Remove-ResourceGroup. One of the screenshots wrongly contains "Remove-ResourceGroups" (plural).

MGoedtel commented:
@alexandair Thanks for bringing this to my attention.  I've updated that plus a few other items I decided to change after re-review of it.  We re-wrote this runbook a few times earlier his week while I was testing/re-writing part of the article, I didn't catch the filename change and my account must not have been working with the latest runbook when I pulled it down.  Anyway, I'm getting ready to submit the PR with new updates.

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @mgoedtel, has been notified to review your proposed change.

MGoedtel commented:
@alexandair I had already updated the items and submitted in a new PR on Friday that went live over the weekend, and submitted an updated PR today to refine some points in the article, which combined also includes changes which addresses what you have in your PR.  I am not going to approve this PR being I have already resolved and is pending release.  It should go live tonight.
#please-close

Rich-Lang committed:
Adding Link in Next Steps for Tutorials

Adding Link/Button to Desktop & SPA tutorials to be consistent with WebApp tutorial.
Updating Next Step description typo in SPA tutorial.

Rich-Lang committed:
Update Image Paths

Updating Desktop WebAPI Tutorial to point to the correct image files.

Rich-Lang committed:
Bold Node.js command prompt

Adding bold to the Node.js part of this steps helps ensure users are not running a standard command prompt.

Rich-Lang committed:
Added missing instructions for Configuring Desktop App

Configuring Desktop app section was blank and only included the step to open the solution.  Missing steps were to update ApiScopes and ApiEndpoint.  Text borrowed and massaged from Step 5 of: https://github.com/Azure-Samples/active-directory-b2c-dotnet-desktop

Of the 7 parts of Step 5, only 6&7 are needed due to the previous parts having been completed as part of previous tutorial.

Rich-Lang committed:
Removing App Settings Change

b2cScopes & webAPI configuration changes are not made until the following Tutorial.

Rich-Lang committed:
Merge pull request #1 from Rich-Lang/tutorialupdates

Removing App Settings Change

Rich-Lang committed:
Removing Cleanup Resource

Quickstart tutorials do not create any Azure AD B2C tenants.  As result, the section on cleaning up any created tenant is not needed.

Rich-Lang committed:
Merge pull request #2 from Rich-Lang/tutorialupdates

Removing Cleanup Resource
